.gitignore

```diff
@@ -26,6 +26,7 @@ build/
 .build/
 lib/
 libraries/
-src/
+src
```

Comment:
the src directory is just a symbolic link to the Marlin source directory but git sees it as a file.